README.md

```diff
@@ -58,6 +58,26 @@ Here is an example command to let Gradle add all dependency artifacts to your `v
 gradle --refresh-dependencies --write-verification-metadata sha256 --write-locks dependencies
 ```
 
+Note that in Gradle projects where dependencies are specified dynamically inside a Gradle task, you need to run the same command above, but replace `dependencies` with the specific Gradle task where dependencies are dynamically specified. 
```

Comment:
I'm not 100% sure if _replacing_ the `dependencies` is always a good idea (depending on lazy evaluation and/or project isolation, this might not resolve everything). I would recommend executing both (or "all relevant") tasks, e.g. `gradle dependencies :sub-project1:compileGroovy :sub-project2:compileTestGroovy`.

Reply:
Yes, I agree! What I tried to convey was that you first run the `dependencies` task, then if you encounter a problem you take the same command, replace `dependencies` with your task and re-run it. 

I'll add a new heading and rephrase everything a bit and see if I can come up with something that is more clear.